### PR TITLE
Add team meeting plan and communication plan

### LINKED ISSUE #21: Team Meeting and Communication Plan
**Report:** Development Plan

### PR DESCRIPTION
**Pull request by TimChoy**
Close #21

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -35,8 +37,23 @@
 
 \section{Team Meeting Plan}
 
+The team will have weekly team meetings, currently set to 10 am every Friday, though it can be 
+flexible and will be subject to change in the winter semester. These meetings will be held on 
+Discord. The purpose of these meetings are to realign the team to the current deliverables and to 
+assign tasks to each team member. An agenda of every team meeting will be written as a Github
```
Comment (arkinmodi) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:43]:
```suggestion
assign tasks to each team member. An agenda of every team meeting will be written as a GitHub 
```

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -35,8 +37,23 @@
...
+Discord. The purpose of these meetings are to realign the team to the current deliverables and to 
+assign tasks to each team member. An agenda of every team meeting will be written as a Github 
+issue, and tagged with the meeting number and meeting date. Each meeting will be archived and can 
+be easily accessed by searching for issues with the ``meeting notes'' tag. Additional meetings with 
+the supervisor will be held on a need to meet basis.
+
 \section{Team Communication Plan}
 
+The team will communicate primarily through Discord, Facebook Messenger, and Github Issues. Discord 
+and Facebook messenger will be used for quick communication between the team members, with the added
```
Comment (arkinmodi) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:51]:
```suggestion
and Facebook Messenger will be used for quick communication between the team members, with the added 
```
Technically its official name is ["Messenger"](https://about.meta.com/technologies/messenger/). Could say Meta's Messenger if we want to super accurate.

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 3]
```diff
@@ -35,8 +37,23 @@
...
+flexible and will be subject to change in the winter semester. These meetings will be held on 
+Discord. The purpose of these meetings are to realign the team to the current deliverables and to 
+assign tasks to each team member. An agenda of every team meeting will be written as a Github 
+issue, and tagged with the meeting number and meeting date. Each meeting will be archived and can 
+be easily accessed by searching for issues with the ``meeting notes'' tag. Additional meetings with 
+the supervisor will be held on a need to meet basis.
+
 \section{Team Communication Plan}
 
+The team will communicate primarily through Discord, Facebook Messenger, and Github Issues. Discord
```
Comment (arkinmodi) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:50]:
```suggestion
The team will communicate primarily through Discord, Facebook Messenger, and GitHub Issues. Discord 
```

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 4]
```diff
@@ -35,8 +37,23 @@
...
+issue, and tagged with the meeting number and meeting date. Each meeting will be archived and can 
+be easily accessed by searching for issues with the ``meeting notes'' tag. Additional meetings with 
+the supervisor will be held on a need to meet basis.
+
 \section{Team Communication Plan}
 
+The team will communicate primarily through Discord, Facebook Messenger, and Github Issues. Discord 
+and Facebook messenger will be used for quick communication between the team members, with the added 
+benefit that our supervisor will also use Discord. Team meetings will also be hosted on Discord. For 
+general communication, organization, and job delegation, Github Issues will be used. We have set up
```
Comment (arkinmodi) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:53]:
```suggestion
general communication, organization, and job delegation, GitHub Issues will be used. We have set up 
```

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 5]
```diff
@@ -35,8 +37,23 @@
...
+be easily accessed by searching for issues with the ``meeting notes'' tag. Additional meetings with 
+the supervisor will be held on a need to meet basis.
+
 \section{Team Communication Plan}
 
+The team will communicate primarily through Discord, Facebook Messenger, and Github Issues. Discord 
+and Facebook messenger will be used for quick communication between the team members, with the added 
+benefit that our supervisor will also use Discord. Team meetings will also be hosted on Discord. For 
+general communication, organization, and job delegation, Github Issues will be used. We have set up 
+Github Projects to further organize our issues into categories such as ``Todo'', ``In Progress'',
```
Comment (arkinmodi) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:54]:
```suggestion
GitHub Projects to further organize our issues into categories such as ``Todo'', ``In Progress'', 
```

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 2 above)
Comment (TimChoy) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:51]: I'll just change it to Messenger. I forgot about that

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**